website/docs/backends/types/s3.html.md

```diff
@@ -163,8 +163,7 @@ The following configuration options or environment variables are supported:
  * `shared_credentials_file`  - (Optional) This is the path to the
    shared credentials file. If this is not set and a profile is specified,
    `~/.aws/credentials` will be used.
- * `token` - (Optional) Use this to set an MFA token. It can also be
-   sourced from the `AWS_SESSION_TOKEN` environment variable.
+ * `token` / `AWS_SESSION_TOKEN` - (Optional) Use this to set an MFA token.
```

Comment:
Changed the wording of how this is sourced to match the other values in this document.